packages/suite/src/components/wallet/TransactionItem/index.tsx

```diff
@@ -124,20 +124,15 @@ const TransactionItem = React.memo(
         );
 
         // join together regular targets, internal and token transfers
-        // ethereum tx has either targets or transfers
-        // cardano tx can have both at the same time
         const allOutputs: (
             | { type: 'token'; payload: (typeof tokens)[number] }
             | { type: 'internal'; payload: (typeof internalTransfers)[number] }
             | { type: 'target'; payload: WalletAccountTransaction['targets'][number] }
-        )[] =
-            transaction.type === 'self'
-                ? [...targets.map(t => ({ type: 'target' as const, payload: t }))]
-                : [
-                      ...targets.map(t => ({ type: 'target' as const, payload: t })),
-                      ...internalTransfers.map(t => ({ type: 'internal' as const, payload: t })),
-                      ...tokens.map(t => ({ type: 'token' as const, payload: t })),
-                  ];
```

Comment:
I don't understand this change. How is it related?

Reply:
it's not, but it's nonsense